plugins/services/src/js/components/forms/ContainerServiceFormSection.js

```diff
@@ -23,92 +23,100 @@ const {DOCKER, NONE, MESOS} = ContainerConstants.type;
     dockerOnly: 'Grant runtime privileges is only supported in Docker Runtime.'
   },
   forcePullImage: {
-    label: <span>Force Pull Image On Launch</span>,
-    helpText: 'Force Docker to pull the image before launching each task.',
+    label: 'Force Pull Image On Launch',
+    helpText: 'Force Docker to pull the image before launching each instance.',
     dockerOnly: 'Force pull image on launch is only supported in Docker Runtime.'
   }
 };
 
-const getArtifactsLabel = () => {
-  return (
-    <FieldLabel>
-      {'Artifact URI '}
-      <Tooltip
-        content="Provided URIs are passed to Mesos fetcher module and resolved at runtime."
-        interactive={true}
-        maxWidth={300}
-        scrollContainer=".gm-scroll-view"
-        wrapText={true}>
-        <Icon color="grey" id="circle-question" size="mini" />
-      </Tooltip>
-    </FieldLabel>
-  );
-};
-
 class ContainerServiceFormSection extends Component {
-  getArtifactsInputs(data = []) {
-    const errors = this.props.errors.fetch || [];
+  getArtifactsLabel() {
+    const tooltipContent = (
+      <span>
+        {'If your service requires additional files and/or archives of files, enter their URIs to download and, if necessary, extract these resources. '}
+        <a
+          href="https://mesosphere.github.io/marathon/docs/application-basics.html"
+          target="_blank">
+          More information
+        </a>.
+      </span>
+    );
 
-    let content = data.map((item, index) => {
-      let label;
+    return (
+      <FieldLabel>
+        {'Artifact URI '}
+        <Tooltip
+          content={tooltipContent}
+          interactive={true}
+          maxWidth={300}
+          scrollContainer=".gm-scroll-view"
+          wrapText={true}>
+          <Icon color="grey" id="circle-question" size="mini" />
+        </Tooltip>
+      </FieldLabel>
+    );
+  }
+
+  getArtifactsInputs() {
+    const {data, errors, path} = this.props;
+    const artifacts = findNestedPropertyInObject(data, 'fetch') || [];
+    const artifactErrors = findNestedPropertyInObject(
+      errors,
+      `${path}.fetch`
+    ) || [];
+
+    if (artifacts.length === 0) {
+      return (
+        <FormRow>
+          <FormGroup className="column-10">
+            {this.getArtifactsLabel()}
+          </FormGroup>
+        </FormRow>
+      );
+    }
+
+    return artifacts.map((item, index) => {
+      let label = null;
       if (index === 0) {
-        label = getArtifactsLabel();
+        label = this.getArtifactsLabel();
       }
 
       return (
-        <FormRow key={index}>
+        <FormRow key={`${path}.artifacts.${index}`}>
           <FormGroup
             className="column-10"
-            showError={Boolean(errors[index])}>
+            showError={Boolean(artifactErrors[index])}>
             {label}
             <FieldInput
               name={`fetch.${index}.uri`}
-              type="text"
+              placeholder="http://"
```

Comment:
This placeholder is confusing. Is it telling the user that their URI must start with `http://`? Or is it telling me that I can exclude the `http://`? Either way let's add some instructional text here.